src/ModuleProcess.cs

```diff
@@ -118,11 +119,16 @@ public class ModuleProcess : IProcessControl {
         private IContainer ConfigureContainer(IConfigurationRoot configuration) {
 
             var config = new Config(configuration);
+            var clientServicesConfiguration = new ClientServicesConfig(configuration);
```

Comment:
Would suggest rolling the configuration interface into the Config class, same as the other config components.   Then you do not need to make changes in here, only in Config.cs

Reply:
done

Reply:
Ah - I am so sorry - I should have been clearer.  Create the ClientServicesConfig in the Config constructor and forward all to the inner object, overriding what you need to override.   But this is ok, we can change that in the future if we ever want to.

Reply:
got it, thanks for the explanation